client/state/posts/actions.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+import {
+	POST_RECEIVE,
+	POSTS_RECEIVE,
+	POSTS_REQUEST,
+	POSTS_REQUEST_SUCCESS,
+	POSTS_REQUEST_FAILURE
+} from 'state/action-types';
+
+/**
+ * Returns an action object to be used in signalling that a post object has
+ * been received.
+ *
+ * @param  {Object} post Post received
+ * @return {Object}      Action object
+ */
+export function receivePost( post ) {
+	return {
+		type: POST_RECEIVE,
+		post
```

Comment:
It seems like we could prevent some duplication of code and error if we simply called the multi-post action `POSTS_RECEIVE` here, passing in `[ post ]` instead of `post`